solutions/functions/sum-of-multiples.js

```diff
@@ -0,0 +1,36 @@
+let fs = require("fs");
+let data = fs.readFileSync(0, 'utf-8');
+let idx = 0;
+data = data.split('\n');
+
+function readLine() {
+  idx++;
+  return data[idx - 1];
+}
+
+// -------- Do NOT edit anything above this line ----------
+
+//Write  your function here
+
+function sum_of_multiples(list, multiple){
```

Comment:
@Abduliitr 
1. Please update the function name to `camelCase`. It is updated in the template.
2. Rename `multiple` to `divisor`.

Reply:
Addressed